packages/safe-tools-client/app/components/schedule-collapse-panel/index.gts

```diff
@@ -0,0 +1,91 @@
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+import { type EmptyObject } from '@ember/component/helper';
+import { svgJar } from '@cardstack/boxel/utils/svg-jar';
+
+import './index.css';
+
+type Info = { 
+  title: string; description: string;
+}
+
+const panelTitle = 'Scheduled crypto payments for your convenience';
+
+const schedulerSteps: Array<Info> = [
+  {
+    title: 'Set up your payment account',
+    description: 'Connect your wallet, choose a blockchain, set up your safe',
+  },
+  {
+    title: 'Schedule your payments',
+    description: 'Specify the date & time for upcoming & recurring transactions',
+  },
+  {
+    title: 'Set up your payment account',
+    description: 'See details of your past & future crypto payments at a glance',
+  },
+];
+
+
+const schedulerFeatures: Array<Info & { iconName: string; }> = [
+  {
+    title: 'Non-Custodial',
+    description: 'Retain full control of your tokens',
+    iconName: 'non-custodial'
+  },
+  {
+    title: 'Reliable',
+    description: 'Be sure that your payments go through',
+       iconName: 'checked'
+  },
+  {
+    title: 'Punctual',
+    description: 'Never miss a payment again',
+       iconName: 'punctual'
+  },
+];
+
+interface Signature {
+  Element: HTMLElement;
+  Args: { 
+    noCollapse?: boolean
+    };
+  Blocks: EmptyObject;
```

Comment:
I've been told that we can leave `Blocks` off in this situation, rather than specifying it as `EmptyObject`